beacon-chain/blockchain/receive_block.go

```diff
@@ -468,6 +468,9 @@ func (s *Service) validateStateTransition(ctx context.Context, preState state.Be
 	stateTransitionStartTime := time.Now()
 	postState, err := transition.ExecuteStateTransition(ctx, preState, signed)
 	if err != nil {
+		if ctx.Err() != nil {
+			return nil, err
```

Comment:
do we not need a log or error wrapper for context?

Reply:
it will be logged by the p2p package.